demos/frp/testdata/manifests/client/peer1/frpc-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: frpc-config
+  namespace: clusterlink-system
+data:
+  frpc.toml: |
+    # Set server address
+    serverAddr = "${FRP_SERVER_IP}"
+    serverPort = 30444
+
+    [[proxies]]
+    name = "clusterlink-peer1"
+    type = "stcp"
+    localIP = "clusterlink.clusterlink-system.svc.cluster.local"
+    localPort = 443
+    secretKey = "abcdefg"
```

Comment:
secret monitoring/scanning on GH might not like these.
Can you use envsubst to fill in the secret on the fly?
Generate 10 alphanumeric with `echo $(date +%s%N) | sha256sum | head -c 10`